opentracing-flowid/opentracing-flowid/src/test/java/org/zalando/opentracing/flowid/DefaultFlowNoopTracerTest.java

```diff
@@ -12,4 +17,11 @@
     void shouldThrowNoActiveSpanFoundException() {
         assertThrows(IllegalStateException.class, unit::currentId);
     }
+
+    @Test
+    void writeToShouldNotThrow() {
+        final Map<String, String> target = new HashMap<>();
+        unit.writeTo(target::put);
+        assertEquals(emptyMap(), target);
+    }
```

Comment:
This test was (before the fix) failing with an IllegalStateException in the `unit.writeTo` line.
Now it just does nothing (as proven by the assert).